ui/selectors/selectors.js

```diff
@@ -2462,6 +2462,32 @@ export const getCurrentNetwork = createDeepEqualSelector(
   },
 );
 
+export const getDappActiveNetwork = createDeepEqualSelector(
```

Comment:
This file is scary and with all the imports is a recipe for circular dependency

Possible to have this selector be somewhere else, like close to where it's used for now?